datalake/src/main/java/com/sequenceiq/datalake/flow/dr/restore/DatalakeRestoreFailureReason.java

```diff
@@ -0,0 +1,16 @@
+package com.sequenceiq.datalake.flow.dr.restore;
+
+public enum DatalakeRestoreFailureReason {
```

Comment:
could you pls rename to DatalakeRestoreReason as it is not the reason of the failure but the restore.

Reply:
Corrected. THanks!